roles/matrix-jitsi/tasks/setup_jitsi_prosody.yml

```diff
@@ -25,31 +25,40 @@
     force: "{{ omit if ansible_version.major > 2 or ansible_version.minor >= 8 else matrix_jitsi_prosody_docker_image_force_pull }}"
   when: matrix_jitsi_enabled|bool
 
-- name: Ensure jitsi-prosody environment variables file created
+- name: Ensure jitsi-prosody environment variables file is created
   template:
     src: "{{ role_path }}/templates/prosody/env.j2"
     dest: "{{ matrix_jitsi_prosody_base_path }}/env"
     mode: 0640
   when: matrix_jitsi_enabled|bool
 
-- name: Ensure matrix-jitsi-prosody.service installed
+- name: Ensure matrix-jitsi-prosody.service file is installed
   template:
     src: "{{ role_path }}/templates/prosody/matrix-jitsi-prosody.service.j2"
     dest: "{{ matrix_systemd_path }}/matrix-jitsi-prosody.service"
     mode: 0644
   register: matrix_jitsi_prosody_systemd_service_result
   when: matrix_jitsi_enabled|bool
 
-- name: Ensure systemd reloaded after matrix-jitsi-prosody.service installation
+- name: Ensure systemd service is reloaded after matrix-jitsi-prosody.service installation
   service:
     daemon_reload: yes
   when: "matrix_jitsi_enabled and matrix_jitsi_prosody_systemd_service_result.changed"
 
+
+- name: Ensure authentication is properly configured
+  include_tasks:
+    file: "{{ role_path }}/tasks/util/setup_jitsi_auth.yml"
+  when:
+    - matrix_jitsi_enabled|bool
+    - matrix_jitsi_enable_auth|bool
```

Comment:
Thanks for the advice. I'll make the length alteration, as I feel that's probably better than various "if" checks.

Regarding the file name, I kept it that way due to the short length of the command, and so that other auth configurations could be handled in the same file instead of creating one for each.

After finishing this PR, I was hoping to configure OpenID and Matrix Verification in the same file. If you prefer it separate, though, it shouldn't be an issue.

Reply:
I'm going to add the condition the the "setup_jitsi_auth.yml" to keep the structure as I described.

If you'd like me to separate different auth configurations into different files, let me know and I'll go ahead and do that.

Reply:
Sure! If you'd like to allow `setup_jitsi_auth.yml` to be used for more than just *internal* auth, then you'd need to add the `if` checks inside  `setup_jitsi_auth.yml`.

I'd probably just have a dedicated file per auth mechanism, because it's simpler and cleaner (`if` checks in just 1 place; shorter/simpler code in each such `_auth.yaml` file), but you can handle multiple auth mechanisms in the same `setup_jitsi_auth.yml`, if that seems better to you. I don't mind.